puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_bastion_assets_precompile.yaml

```diff
@@ -0,0 +1,64 @@
+- job:
+    name: test_bastion_assets_precompile2
+    project-type: matrix
+    description: 'Run Bastion assets precompile to ensure assets can be generated properly for production.'
+    logrotate:
+      daysToKeep: 30
+      numToKeep: -1
+    builders:
+      - bastion_assets_precompile
+    parameters:
+      - string:
+          name: branch
+          default: master
+          description: 'The branch to build against.'
+      - string:
+          name: pr_number
+          description: 'Katello GitHub PR number to merge, or a URL to a patch file to apply directly'
+      - string:
+          name: pr_git_url
+          description: 'Katello Git URL to PR to test, e.g. git://github.com/user/foo.git'
+      - string:
+          name: pr_git_ref
+          description: 'Katello Git ref for PR to test, usually the branch name.'
+      - string:
+          name: foreman_pr_git_url
+          description: 'Foreman Git URL to PR to test, e.g. git://github.com/user/foo.git'
+      - string:
+          name: foreman_pr_git_ref
+          description: 'Foreman Git ref for PR to test, usually the branch name.'
+      - string:
+          name: base_foreman_branch
+          default: develop
+    scm:
+      - git:
+          url: https://github.com/Katello/bastion
+          branches:
+            - ${branch}
+          basedir: plugin
+          per-build-tag: true
+    axes:
+      - axis:
+          type: user-defined
+          name: rubygems
+          values:
+            - 2.2
+    wrappers:
+      - timeout:
+          elastic-percentage: 200
+          elastic-number-builds: 3
+          timeout: 60
+          fail: true
+          type: elastic
+    publishers:
+      - post-tasks:
+        - matches:
+          - log-text:
+          script: |
+            #!/bin/bash
+            # Clean gemset and database
+            . /etc/profile.d/rvm.sh
+            gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
+            rvm use ruby-${ruby}@${gemset}
+            rvm gemset delete ${gemset} --force
+            exit 0
```

Comment:
I think there is an existing publisher for doing this for us.

Reply:
gotcha, I'll find that